tests/php/test-uninstall.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Test cases for uninstall.php
+ *
+ * @package AMP
+ */
+
+class Test_Uninstall extends WP_UnitTestCase {
```

Comment:
You might want/need to run this test in a separate process since you are requiring some files and defining constants. Don't want that to leak to other tests.

Reply:
@swissspidy I have addressed it. Please review it.